daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -328,7 +332,7 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments)
   object parse extends scallop.Subcommand("parse") {
     banner("""|Usage: daffodil parse (-s <schema> [-r [{namespace}]<root>] [-p <path>] |
               |                       -P <parser>)
-              |                      [--validate [mode]]
+              |                      [--validate [mode]] [--validate_output file]
```

Comment:
Please use brackets around "file" for consistency with the other usage lines: `[--validate_output <file>]`